### PR TITLE
[Docs] Fix typos 

### PR DESCRIPTION
**Pull request by thisiskaransgit**
Signed-off-by: Karan Singh Thakur <karanthakur.work@gmail.com>

The scope of the changes included misspelling and fixing grammar mistakes.

This PR fixes  typos and grammatical errors found in 

1. ADOPTERS.md
2. Contributing.md
3. ui/README.md

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -11,9 +11,12 @@
...
 - [BootLabs](https://www.bootlabs.in/)
-  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud.  Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Micoservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the imact meshery has done for the Mesh deployment and Management and reduced the time to market "
+  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud. Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Microservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the impact meshery has done for the Mesh deployment and Management and reduced the time to market "
```
Comment (leecalcote) [inline ADOPTERS.md:14]: Please add whitespace between Service and Mesh. Please capitalize every reference to Meshery.

Comment (leecalcote) [reply ADOPTERS.md:14]: Add punctuation on the end of the sentence.

Add "in" to "Meshery comes handy"

Comment (thisiskaransgit) [reply ADOPTERS.md]: Implemented all the requested changes.

File: ADOPTERS.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 - [Intel](https://www.intel.com)
-  - "We are working on the acceleration and security of open source Service Mesh projects. Meshery is the best tool for managing and operating different Service Mesh projects, it greatly helped us compare the functions and performance of them, and demonstrate the results of our efforts. We also love Meshery's community where we can connect with Service Mesh talents and enthusiasts from all over the world!"
+  - "We are working on the acceleration and security of open-source Service Mesh projects. Meshery is the best tool for managing and operating different Service Mesh projects, it greatly helped us compare the functions and performance of them, and demonstrate the results of our efforts. We also love Meshery's community where we can connect with Service Mesh talents and enthusiasts from all over the world!"
```
Comment (leecalcote) [inline ADOPTERS.md:16]: Don't hyphenate "open source".

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ADOPTERS.md [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
 - [BootLabs](https://www.bootlabs.in/)
-  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud.  Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Micoservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the imact meshery has done for the Mesh deployment and Management and reduced the time to market "
+  - "We are a true Boutique Tech Consulting partner who understands the pain points of the all-sized company in terms of embracing Cloud. Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers, the cutting-edge technology and the trend is Service Mesh now. We enable our Clients to make their Microservices more secure with Mesh. When we talk about Service Meshes, Meshery comes in handy. We have seen the impact Meshery has done for the Mesh deployment and Management and reduced the time to market. "
```
[the remark this answers is not in the document]
Comment (thisiskaransgit) [reply ADOPTERS.md]: Done.

File: ADOPTERS.md (hunk 2 above)
Comment (thisiskaransgit) [reply ADOPTERS.md]: Done.

File: ADOPTERS.md [hunk 4]
```diff
@@ -1,19 +1,22 @@
...
 - [BootLabs](https://www.bootlabs.in/)
-  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud.  Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Micoservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the imact meshery has done for the Mesh deployment and Management and reduced the time to market "
+  - "We are a true Boutique Tech Consulting partner who understands the pain points of the all-sized company in terms of embracing Cloud. Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers, the cutting-edge technology and the trend is Service Mesh now. We enable our clients to make their microservices more secure with Mesh. When we talk about service meshes, Meshery comes in handy. We have seen the impact Meshery has done for the Mesh deployment and Management and reduced the time to market. "
```
Comment (leecalcote) [inline ADOPTERS.md:14]: Everything except Meshery and the company name should be lowercased.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ADOPTERS.md (hunk 4 above)
Comment (thisiskaransgit) [reply ADOPTERS.md]: Okay, I misinterpreted the last comment, so I lowercased everything except "Meshery" and Company's name.
I was hesitant about changing "Service Mesh" as it was mentioned in uppercase while quoting the other adopters.

Comment (leecalcote) [reply ADOPTERS.md]: That logic makes sense to me. The other quotes shouldn't have capitalized "Service Mesh" either, though.

Comment (leecalcote) [reply ADOPTERS.md]: Same thing for stuff like "Global Scale" - that shouldn't be capitalized either.

File: ADOPTERS.md [hunk 5]
```diff
@@ -1,19 +1,22 @@
...
 - [Intraserve Systems](http://intraservesystems.com)
-  - [Mohsen Houshmand](https://twitter.com/houshym): "Working with service mesh technology, I find one of the most useful tools is Meshery! I use Meshery to run POCs for our customers to help them adopt, and then, ongoing to check our mesh configuration. The most fantastic part about Meshery is it's performance management features."
+  - [Mohsen Houshmand](https://twitter.com/houshym): "Working with Service Mesh technology, I find one of the most useful tools is Meshery! I use Meshery to run POCs for our customers to help them adopt, and then, ongoing to check our mesh configuration. The most fantastic part about Meshery is its performance management features."
```
Comment (leecalcote) [inline ADOPTERS.md:6]:
```suggestion
  - [Mohsen Houshmand](https://twitter.com/houshym): "Working with service mesh technology, I find one of the most useful tools is Meshery! I use Meshery to run POCs for our customers to help them adopt, and then, ongoing to check our mesh configuration. The most fantastic part about Meshery is its performance management features."
```

File: ADOPTERS.md [hunk 6]
```diff
@@ -1,19 +1,22 @@
 - [HPE Labs- Cloudless](https://www.labs.hpe.com/page/cloudless)
-  - "Our Complex, Dynamic, and Intelligent system demands leverage different tools to efficiently manage service exchange and ecosystem at Global Scale; for instance, we need to identify right fit Service Mesh to address varying workload scenarios, distributed systems across locations, different security tiers, minimal latency etc. Meshery helps us curate Service Mesh for tailored need by helping us running simulations and identifying the right fit Service Mesh that works efficiently across diverse ecosystem of tools."
+  - "Our Complex, Dynamic, and Intelligent system demands leverage different tools to efficiently manage service exchange and ecosystem at a Global Scale; for instance, we need to identify the right fit Service Mesh to address varying workload scenarios, distributed systems across locations, different security tiers, minimal latency etc. Meshery helps us curate Service Mesh for tailored need by helping us running simulations and identifying the right fit Service Mesh that works efficiently across a diverse ecosystem of tools."
```
Comment (leecalcote) [inline ADOPTERS.md:2]:
```suggestion
  - "Our complex, dynamic, and intelligent system demands leverage different tools to efficiently manage service exchange and ecosystem at a global scale; for instance, we need to identify the right fit service mesh to address varying workload scenarios, distributed systems across locations, different security tiers, minimal latency etc. Meshery helps us curate service mesh for tailored need by helping us running simulations and identifying the right fit service mesh that works efficiently across a diverse ecosystem of tools."
```

**Comment by leecalcote** [comment]
Thank you, @thisiskaransgit! 👍

**Pull request merged**